graph/edge/impl/TypeEdgeImpl.java

```diff
@@ -418,6 +403,16 @@ public TypeVertex to() {
             return to;
         }
 
+        @Override
+        VertexIID.Type fromIID() {
```

Comment:
we introduce this method here in order to avoid looking up a cached `from()` in order to get the IID for the equality function.